packages/@dcl/inspector/src/components/Tree/Tree.css

```diff
@@ -1,39 +1,70 @@
-.tree {
-  background-color: #ffffff;
-  overflow: scroll;
-  height: 100vh;
-  width: 20rem;
-  border-right: 0.1rem solid #dedede;
+.Hierarchy {
```

Comment:
One component's style should not be dependant on other's. The `Tree` component should work even when not used within `Hierarchy`. Let's change this to be `.Tree` instead (and lets add the `Tree` class to the components outter div).